orchestra/models/core/models.py

```diff
@@ -592,6 +592,9 @@ class TodoListTemplate (TodoListTemplateMixin, BaseModel):
             if it is generated by the system.
         todos (str)
             A JSON blob that describe the todos in the todo list template.
+        conditional_property_function (str)
```

Comment:
explain the structure---is it a list of conditions that get ORed together, with properties in each dictionary getting ANDed, or the other way?